gems/codders-dataset/OSVDB-108582.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Should this not be combined into 1 with OSVDB-108583 as they are the same CVE? Opinion?